src/main/java/frc/robot/commands/RotatePivotCommand.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import java.util.function.DoubleSupplier;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Pivot;
+
+public class RotatePivotCommand extends Command {
```

Comment:
It can also be helpful to add Javadoc that describes the entire command, eg. "`Rotates the {@link Pivot} to a given angle or dynamically supplied angle until the command is interrupted`"